tests/fixtures/abridged_localpart_invalid.txt

```diff
@@ -39,7 +37,6 @@ first..last
 "\"
 abc\
 Doug\ \"Ace\"\ Lovell
-"Doug "Ace" L."
```

Comment:
Valid address because we allow `Foo foo@bar.com` without `<>`. Equivalent to `"Doug "Ace " L."@example.com` where `"Doug "Ace` is the display name.